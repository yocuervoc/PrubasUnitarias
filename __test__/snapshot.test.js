import {getCharacter} from "../snapshot"
import rick from '../rick.json'

describe("es hora de la snapshot", ()=>{
    test(' test de snapshot', ()=>{
        expect(getCharacter(rick)).toMatchSnapshot()
    })
    test("excepciones de snapshot ", ()=>{

        const user={
            id: Math.floor(Math.random() * 20),
            name: "Yonatan Cuervo"
        }
        expect(user).toMatchSnapshot({
            id:expect.any(Number)
        })
    })
})
