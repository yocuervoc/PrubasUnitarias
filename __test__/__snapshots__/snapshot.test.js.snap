// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`es hora de la snapshot  test de snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;

exports[`es hora de la snapshot excepciones de snapshot  1`] = `
Object {
  "id": Any<Number>,
  "name": "Yonatan Cuervo",
}
`;
